### PR TITLE
qa: add a method to get keyring for a client_id

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
The method wraps very frequently used code-phrase and adds check to
avoid potential bugs.

## Checklist
[x] QA improvement

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by nmshelke** [comment]
Are you only adding method (without actual usage) ?

**Review by nmshelke — COMMENTED** [review]
Are you only adding method (without actual usage) ?

**Comment by vshankar** [comment]
@rishabh-d-dave mentioned that this is required for "fs authorize" pr - https://github.com/ceph/ceph/pull/41779

Better to include these changes in that pr.

**Comment by rishabh-d-dave** [comment]
For easy of review, testing and merge this commit is being moved again.

**Comment by rishabh-d-dave** [comment]
For easy of review, testing and merge this commit is being moved again.

**Comment by rishabh-d-dave** [comment]
jenkins test make check

File: qa/tasks/vstart_runner.py [hunk 1]
```diff
@@ -762,6 +762,23 @@ def find_remote(self, daemon_type, daemon_id):
         """
         return LocalRemote()
 
+    def get_keyring(self, client_id):
```
Comment (vshankar) [inline qa/tasks/vstart_runner.py:765]: Can this be inherited via ceph_manager.py?

Comment (rishabh-d-dave) [reply qa/tasks/vstart_runner.py]: Yes, this was supposed to updated after my "inherit" got merged.

Done, now. Thanks.

**Review by vshankar — APPROVED** [review]

**Comment by rishabh-d-dave** [comment]
https://tracker.ceph.com/projects/cephfs/wiki/Main#2022-Jul-15

**Pull request merged**